docs/src/further_topics/ugrid/data_model.rst

```diff
@@ -0,0 +1,361 @@
+   :align: center
+   :width: 300
+
+   Every mesh node is completely independent
+
+* Faces and volumes can have variable node counts, i.e. different numbers of
+  sides. This is achieved by masking the unused 'slots' in the connectivity array.
+
+.. figure:: images/ugrid_variable_faces.svg
+   :alt: Diagram demonstrating mesh faces with variable node counts
+   :align: center
+   :width: 300
+
+   Mesh faces can have different node counts (using masking)
+
+* Data can be assigned to lines (edges) just as easily as points (nodes) or
+  areas (faces).
+
+.. figure:: images/ugrid_edge_data.svg
+   :alt: Diagram demonstrating data assigned to mesh edges
+   :align: center
+   :width: 300
+
+   Data can be assigned to mesh edges
+
+.. _ugrid implications:
+
+What does this mean?
+--------------------
+Meshes can represent much more varied spatial arrangements
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The highly specific way of recording position (geometry) and shape
+(topology) allows meshes to represent essentially **any** spatial arrangement
+of data. There are therefore many new applications that aren't possible using a
+structured grid, including:
+
+* `The UK Met Office's LFRic cubed-sphere <https://hps.vi4io.org/_media/events/2018/sig-io-uk-adams.pdf>`_
+* `Oceanic model outputs <https://doi.org/10.3390/jmse2010194>`_
+
+.. todo:
+        a third example!
+
+Mesh 'payload' is much larger than with structured grids
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Coordinates are recorded per-node, and connectivities are recorded per-element.
+This is opposed to a structured grid, where a single coordinate value is shared
+by every data point/area along that line.
+
+For example: representing a cubed-sphere using a mesh leads to coordinates and
+connectivities being **~8 times larger than the data itself**, as opposed to a
+small fraction of the data size when dividing a sphere using a structured grid.
+
+This further increases the emphasis on lazy loading and processing of data
+using packages such as Dask.
+
+.. note::
+
+        The large, 1D data arrays associated with meshes are a very different
+        shape to what Iris users and developers are used to. It is suspected
+        that optimal performance will need new chunking strategies, but at time
+        of writing (``Jan 2022``) experience is still limited.
+
+.. todo:
+        Revisit when we have more information.
+
+Spatial operations on mesh data are more complex
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Detail: :doc:`operations`
+
+Indexing a mesh data array cannot be used for:
+
+#. Region selection
+#. Neighbour identification
+
+This is because - unlike with a structured data array - relative position in
+a mesh's 1-dimensional data arrays has no relation to relative position in
+space. We must instead perform specialised operations using the information in
+the connectivities present, or by translating the mesh into a format designed
```

Comment:
I may be reading this incorrectly, but I would have thought this should be:

> We must instead perform specialised operations using the information **present** in the connectivities

Reply:
I was actually thinking that if there were more connectivities than the bare minimum (e.g. `face_edge_connectivity`), it would be possible to write something that quickly finds neighbours or maybe identifies regions.

This will probably do:

```suggestion
the mesh's connectivities, or by translating the mesh into a format designed
```